<!-- -->
<launch>

<!-- global params -->
    
<node name="airsim_node" pkg="airsim_ros_pkgs" type="airsim_node" output="screen">

    <!-- ROS timer rates. Note that timer callback will be processed at maximum possible rate, upperbounded by the following ROS params -->
    <param name="update_airsim_img_response_every_n_sec" type="double" value="0.01" /> 
    <param name="update_airsim_control_every_n_sec" type="double" value="0.01" />

    <!-- Camera calibration -->
    <!-- todo add relative path -->
    <!-- ubuntu -->
    <!-- <param name="front_left_calib_file" type="string" value="/home/madratman/projects/airsim_roscpp_ws/src/airsim_roscpp_pkgs/calib/front_left_720x1080.yaml" /> -->
    <!-- <param name="front_right_calib_file" type="string" value="/home/madratman/projects/airsim_roscpp_ws/src/airsim_roscpp_pkgs/calib/front_right_720x1080.yaml" /> -->
    <!-- <param name="front_left_calib_file" type="string" value="/home/madratman/projects/airsim_roscpp_ws/src/airsim_roscpp_pkgs/calib/front_left_480x640.yaml" /> -->
    <!-- <param name="front_right_calib_file" type="string" value="/home/madratman/projects/airsim_roscpp_ws/src/airsim_roscpp_pkgs/calib/front_right_480x640.yaml" /> -->
    <!-- windows -->
    <!-- <param name="front_left_calib_file" type="string" value="/mnt/c/Users/ramadaan/projects/AirSim_wsl/airsim_roscpp_ws/src/airsim_ros_pkgs/calib/front_left_720x1080.yaml" /> -->
    <!-- <param name="front_right_calib_file" type="string" value="/mnt/c/Users/ramadaan/projects/AirSim_wsl/airsim_roscpp_ws/src/airsim_ros_pkgs/calib/front_right_720x1080.yaml" /> -->
    <param name="front_left_calib_file" type="string" value="/mnt/c/Users/ramadaan/projects/AirSim_wsl/airsim_roscpp_ws/src/airsim_ros_pkgs/calib/front_left_480x640.yaml" />
    <param name="front_right_calib_file" type="string" value="/mnt/c/Users/ramadaan/projects/AirSim_wsl/airsim_roscpp_ws/src/airsim_ros_pkgs/calib/front_right_480x640.yaml" />

</node>

</launch>

